File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        d={}
        for i in nums:
            d[i]=d.get(i,0)+1
        print(d)
        res=[]

        for _ in range(k):
            maxx = -1
            max_i = 0
            for i in d.keys():
                if d[i] > maxx:
                    maxx = d[i]
                    max_i = i 
            res.append(max_i)
            d[max_i] = -1
        return res
            
        

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        Arrays.sort(nums);
        if(nums.length==1 || nums.length==0)
            return nums.length;
        if(nums.length==2)
        {
            if(nums[0]==nums[1])
                return 1;
            return (nums[0]+1==nums[1]) ? 2 : 1;
        }
        int max=1; int maxx=1;
        for(int i=1; i<nums.length; i++)
        {
            if(nums[i]==nums[i-1]+1)
            {
                maxx=maxx+1;
                if(maxx > max)
                    max=maxx;
            }
            else if(nums[i]==nums[i-1])
            {
                continue;
            }
            else
            {
                maxx=1;
            }
        }
        return max;
    }   
}